### ISSUE TITLE
/otp <offlineplayer> <otherplayer> and /otp <offlineplayer> <x> <y> <z> cause CTD on join

### ISSUE THREAD

**Issue by downslope7** [hilburn/OTP#2]
Crash log [here](https://gist.github.com/downslope7/aa2613d1e70c1693b980).

**Comment by hilburn** [comment]
Does `/otp <player1> x y z` throw the same issue when player 1 is offline and then joins?

**Comment by downslope7** [comment]
Yeah. [Here's an updated crash log](https://gist.github.com/downslope7/24fb3b8f387c10d8ff29) with both server and client side logs if that helps. Looks like it is a cross-mod interaction with WR-CBE.

**Comment by hilburn** [comment]
Ok - I'll try to fix this if possible this evening
